core/app/cview.class.php

```diff
@@ -27,7 +27,7 @@ public function __construct()
     private function init()
     {
         // Set to true to force template generation if a template has changed
-        $this->compile_check = false;
+        $this->compile_check = $config['compile_check'];
```

Comment:
Problem: This variable is not defined in config.php and at least, you should check if the variable exist and not null before using it.
I won't use this change anyway as I prefer to use this class property for the moment.
A "debug mode" which include more than that will come in the future